src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -85,8 +87,8 @@ public static class PrereleaseCompatibilityTransformer
                             modifiedLog |= UpdateSarifLogVersionAndSchema(logObject);
                             break;
                         }
-                        case "http://json.schemastore.org/sarif-2.1.0-rtm.5":
-                        case "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json":
+                        case "http://json.schemastore.org/sarif-2.1.0-rtm.6":
+                        case "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json":
                         {
                             // Current schema version. There should be no work to do
```

Comment:
>Current [](http://example.com/codeflow?start=31&length=7)

So, we should add a prerelease transformer test for this scenario, updating from rtm.5 to current. Looks like a no-op for now.